glyphs2fontir/src/source.rs

```diff
@@ -804,7 +804,7 @@ impl Work<Context, WorkId, Error> for KerningInstanceWork {
                 Some(((side1, side2), pos_adjust))
             })
             .for_each(|(participants, (_, value))| {
-                *kerning.kerns.entry(participants).or_default() = (value as f32).into();
+                *kerning.kerns.entry(participants).or_default() = (value.0 as f32).into();
```

Comment:
In the spirit of f64 all the things I wonder if we shoudl file an issue to get rid of the `as f32`

Reply:
I started out assuming that was the PR I was writing until I got in here and noticed there was a clearer root cause. I do  think getting rid of f32 makes sense, though.